package.json

```
@@ -66,11 +67,13 @@
     "eslint-plugin-preact": "^0.1.0",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-simple-import-sort": "^7.0.0",
+    "husky": "7.0.4",
```

```suggestion
    "husky": "^7.0.4",
```
altho i'd suggest removing it, all deps should always use `^`